### PR TITLE
Do not inherit pod name by plugins machines

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Do not inherit pod name by plugins machines.
**Before:**
Screenshot_20190424_104341

**After:**
Screenshot_20190424_103112

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13095

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Review by metlos — APPROVED** [review]

**Comment by sleshchenko** [comment]
ci-test

**Comment by sleshchenko** [comment]
ci-build

**Review by garagatyi — APPROVED** [review]

File: infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplierTest.java [hunk 1]
```diff
@@ -264,6 +262,24 @@ public void shouldNotFillInWarningIfChePluginHasMultiplyContainersAndThereAreNot
     assertTrue(internalEnvironment.getWarnings().isEmpty());
   }
 
+  @Test
+  public void shouldUsePluginNameAndContainerNameForForMachinesNames() throws Exception {
```
Comment (sunix) [inline infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplierTest.java]: ...ForFor... ?

Comment (sleshchenko) [reply infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplierTest.java]: It's my favorite word :smile:

**Comment by sunix** [comment]
Just curious, how are you ending up with a user container named `ws/something` ?, creating from a devfile my user containers (type dockerimage) are not prefixed with `ws/`

**Review by sunix — APPROVED** [review]

**Comment by sleshchenko** [comment]
> Just curious, how are you ending up with a user container named ws/something ?, creating from a devfile my user containers (type dockerimage) are not prefixed with ws/

Not sure if I fully understand your question. Alias or generated value based on docker image is used as machine name of dockerimage component https://github.com/eclipse/che/blob/09ccc0db2636bef5407040b850e51dc6a57ba2a8/wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/component/dockerimage/DockerimageComponentToWorkspaceApplier.java#L113-L114

**Review by amisevsk — APPROVED** [review]
LGTM

**Comment by artaleks9** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1739/) doesn't show any regression against this Pull Request.

**Pull request merged**